src/services/firebase.js

```diff
@@ -1,12 +1,12 @@
 import firebase from 'firebase'
 
 var config = {
-  apiKey: 'AIzaSyC14hJ31aB7YF_0_yOZ_3KG1sPmKlEtLh0',
-  authDomain: 'glean-tennessee.firebaseapp.com',
-  databaseURL: 'https://glean-tennessee.firebaseio.com',
-  projectId: 'glean-tennessee',
-  storageBucket: 'glean-tennessee.appspot.com',
```

Comment:
These values are for the FB project that is administered by Code For Nashville. I think we want to keep these.